control/state.py

```diff
@@ -303,8 +303,8 @@ def unlock_omap(self):
             self.is_locked = False
         except rados.ObjectNotFound as ex:
             self.logger.warning(f"No such lock, the lock duration might have passed")
-        except Exception as ex:
-            self.logger.error(f"Unable to unlock OMAP file: {ex}")
+        except Exception:
+            self.logger.exception(f"Unable to unlock OMAP file")
```

Comment:
Should we raise, instead of pass, wdyt?

Reply:
@baum I didn't want to crash the gateway because the OMAP lock. In the worst case it will behave as it did before the change.